rpc/lib/server/handlers.go

```diff
@@ -538,7 +538,7 @@ func (wsc *wsConnection) writeRoutine() {
 				return
 			}
 		case msg := <-wsc.writeChan:
-			jsonBytes, err := json.Marshal(msg)
+			jsonBytes, err := json.MarshalIndent(msg, "", "  ")
```

Comment:
Here's a nice article about lots on api usability, where I got this idea.  Linking to one section, but it is nice to take a look at the whole thing...

http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#pretty-print-gzip